content/en/blog/2025/opentelemetry-logging-and-you.md

```diff
@@ -0,0 +1,91 @@
+
+## Definitions
+
+Let’s get started with a basic definition of how OpenTelemetry thinks about
+logs. Broadly, logs are any telemetry that is emitted through a LoggerProvider,
+and are created by calling the Logging API. There are two ways we intend for
+users to do this –
+
+- Use the logging API as a sink for existing loggers (sending existing logs to
+  the OpenTelemetry SDK)
+- Use the logging API to emit events, our vision of a structured logging format
+
+Regardless of the type of log, there is one thing they all have in common – logs
+are part of the OpenTelemetry Context. This means that logs will have a
+reference to the context of the transaction or request that they were emitted
+under, irrespective of type or source. This is a fundamental part of the
+OpenTelemetry data model – all of your telemetry must be linked, contextually,
+to other telemetry.
+
+With that said, what’s the distinction between a log and an event? It is rather
+straightforward – events are logs that
+[OpenTelemetry can make guarantees about](/docs/specs/otel/logs/data-model/#events).
+An Event has a mandatory name and a structure that is defined via a schema,
+similar to how semantic conventions are defined today. Some people may refer to,
+or think of, these as ‘structured logs’ and it wouldn’t be wrong. We are using
+the word event because it most clearly describes what an event is – something
+that happens, without a duration, and can be named.
+
+**We believe that all logs should be events**, or at least the overwhelming
+majority of them.
+
+## How is this different from other signals?
+
+Logs and Spans are often thought of as being conceptual twins, especially when
+you start talking about events. What is a span other than an event with a
+particularly detailed schema, after all? A key distinction between them is that
+**spans have durations**. Events make no guarantees or claims about duration or
+time; They could represent a truly instantaneous occurrence, or the result of
+minutes or hours of computation. The other big difference is that **spans have
+an explicit hierarchy**. A span is part of a trace that contains many spans. A
+span with no children is, in and of itself, a complete trace. Events do not have
+this guarantee.
+
+Logs and Metrics differ in more clearly conceptual ways – a metric is a
+numerical series of values across time. While it’s possible to convert logs into
+metrics, this is downstream of the definition of the telemetry itself.
+
+Fundamentally, OpenTelemetry is built on the concept that all signals are
+interpreted together, rather than separately. Events can be considered to be a
+part of the span that they were emitted under, and can be used to represent
+everything from an exception, to security data, to an access log, to timing
+information, and much more. Events are not only useful in the presence of a
+span, but if they’re emitted with a span context, they can reasonably be
+interpreted as contained within the span.
```

Comment:
> Events are not only useful in the presence of a
span, but if they’re emitted with a span context, they can reasonably be
interpreted as contained within the span.

Just some wordsmithing comment: it sounds like both parts separated with 'but' talk about the same - events being useful along with a span, was it the intention?